src/Tomcat/TomcatController.ts

```diff
@@ -264,6 +245,54 @@ export class TomcatController {
         this._tomcatModel.saveServerListSync();
     }
 
+    private async validateWebappPath(webappPath: string): Promise<boolean> {
```

Comment:
Since the return value is `boolean` how about change the name to `isWebAppPathValid()`